### PR TITLE
Consolidate web-eid packages, ignore Arch's version

### PR DESCRIPTION
**Pull request by klemensn**

### CODE REVIEW

File: 800.renames-and-merges/w.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@
 - { setname: wcalc,                    name: w-calc }
 - { setname: wcmcommander,             name: walcommander } # old name
 - { setname: wcmcommander,             name: wcm-commander }
+- { setname: web-eid,                  namepat: "web-eid-(app|native|chrome|firefox)" }
```
[the remark this answers is not in the document]
Comment (klemensn) [reply 800.renames-and-merges/w.yaml]: > * `web-eid-chrome` and `web-eid-firefox` do not exist. See the [list](https://repology.org/projects/?search=web-eid-). You've probably meant `web-eid-webextension`.

Ah, I mixed up Repology project names with Arch/OpenBSD package names.
See the commit message for more details.

>     * Please list names with `name:[]` here. Regexps are computationally heavy and are not really justified for a small bunch of fixed names.

Sure.

>     * Also please add `addflavor: true` to indicate these are different flavors of a project.

Those are not flavours, they're are components that belong together, but may be installed separately.
- `web-eid-native` is the native messaging host used by all browsers;  can also be installed alone for CLI usage
- `web-eid-chrome` provides the Chromium extension and/or configuration pointing at `web-eid-native` system's packages binary
- `web-eid-firefox` same as above, but for Firefox

[the remark this answers is not in the document]
Comment (AMDmi3) [reply 800.renames-and-merges/w.yaml]: These are flavors for repology. Please add `addflavor: true`.

[the remark this answers is not in the document]
Comment (klemensn) [reply 800.renames-and-merges/w.yaml]: Understood, thanks for your input!

**Comment by AMDmi3** [comment]
Just to be sure, https://repology.org/project/web-eid-webextension/versions remains separate project, right?

**Comment by klemensn** [comment]
> Just to be sure, https://repology.org/project/web-eid-webextension/versions remains separate project, right?

Yes.

**Pull request merged**